input.cc

```diff
@@ -2378,53 +2378,101 @@ void time_init(void)
   globals::time_step = (struct time *) malloc((globals::ntstep + 1) * sizeof(struct time));
 
   /// Now set the individual time steps
-  for (int n = 0; n < globals::ntstep; n++)
+  if (LOG_TIMESTEPS)
```

Comment:
Update to handle the enumeration in artisoptions.h. Maybe use a switch statement?